.cspell.yml

```diff
@@ -144,6 +144,7 @@ words:
   - Sharma
   - shippingservice
   - shortcode
+  - Skyscanner
```

Comment:
put Skycanner in line1 of the adopters.yaml, as long as this is not used across multiple pages we keep this local

Reply:
Ah! I see! I completely missed the cspell comment. Will do now.